addons/smtp-mail.sh

```diff
@@ -238,19 +236,19 @@ $(grep -v password /etc/msmtprc)
 Best regards
 The NcVM team
 https://nextcloudvm.com" \
-| mail -s "Test email from your NcVM" "$RECIPIENT" &>/dev/null
+| mail -s "Test email from your NcVM" "$RECIPIENT" > $VMLOGS/mail_msmtp.log 2>&1 # But this works
```

Comment:
this way we rewrite any output to the log file

Reply:
Aah, forgot about that one. Nice catch!

Still would be nice to get msmtp to log everything to that log.

Reply:
The strange thing is that even chmod 666 doesn't work so I really don't know how to fix this.

Reply:
if you don't know either, I would suggest merging this because debugging will be easier for most users (at least for the setup) and solving the permission issue later.

Reply:
This is most likely the problem:
https://askubuntu.com/a/1191200

Reply:
I saw that before, but never tried it. Hence the code in the script:

```
groups msmtp
sudo touch /var/log/msmtp
sudo chown msmtp:msmtp /var/log/msmtp
sudo chmod 660 /var/log/msmtp
```

Please try and see if it works.

Thanks!

Reply:
This works now